### PR TITLE
Add muller to NERSC machines

### PR DESCRIPTION
**Pull request by kngott**
## Summary
Adding a NERSC test machine to the list.

## Checklist

The proposed changes:
- [ ] fix a bug or incorrect behavior in AMReX
- [ ] add new capabilities to AMReX
- [ ] changes answers in the test suite to more than roundoff level
- [ ] are likely to significantly affect the results of downstream AMReX users
- [ ] include documentation in the code and/or rst files, if appropriate

### CODE REVIEW

**Review by jmsexton03 — APPROVED** [review]

File: Tools/GNUMake/Make.machines [hunk 1]
```diff
@@ -34,6 +34,10 @@ ifeq ($(NERSC_HOST), alvarez)
   which_site := nersc
   which_computer := perlmutter
 endif
+ifeq ($(NERSC_HOST), muller)
+  which_site := nersc
+  which_computer := perlmutter
```
Comment (WeiqunZhang) [inline Tools/GNUMake/Make.machines:42]:
```suggestion
  which_computer := muller
```
?

Comment (WeiqunZhang) [reply Tools/GNUMake/Make.machines:42]: Or is it the same as perlmutter?

Comment (kngott) [reply Tools/GNUMake/Make.machines:42]: Muller is like Alvarez, a test machine that should be treated like Perlmutter. So, `which_computer=perlmutter` gives the same configuration without any additional code changes in sites/Make.nersc.

Also, the future of Muller and Alvarez may be to merge them into Perlmutter? So, I don't think anything fancy needs to be done with either of them.

Comment (WeiqunZhang) [reply Tools/GNUMake/Make.machines:42]: I figured that might be the case.  Anyway, could we add a comment?

Comment (kngott) [reply Tools/GNUMake/Make.machines:42]: Sure! How's that one look?

**Review by WeiqunZhang — APPROVED** [review]

**Pull request merged**